src/components/chip/chip.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react"
+import "./chip.scss"
+
+const Chip = () => {
+  return (  
+    <div className={"chip-container"}>
+      <p>"Chip Test"</p>
```

Comment:
Suggestion: We will need to make this dynamic.

In order to place the appropriate text and colors into the component, let's pass props of text and color into component. Alternatively for the color, we can set the color as a condition relative to the tag type inside the component.

Reply:
Sounds good! I'm not quite sure how to make it dynamic with passing text props, so pairing up on this would be great!